nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/CleanSurefireExecution.java

```diff
@@ -120,10 +124,26 @@ public void run() throws MojoExecutionException {
         }
     }
 
-    protected void setupArgline() {
-        Logger.getGlobal().log(Level.FINE, "Running clean surefire.");
+    protected void setupArgline(Xpp3Dom configNode) {
+        String argLineToSet =  this.configuration.toArgLine();
+        boolean added = false;
+        for (Xpp3Dom config : configNode.getChildren()) {
+            if ("argLine".equals(config.getName())) {
+                Logger.getGlobal().log(Level.INFO, "Adding argLine to existing one");
+                String current = config.getValue();
+
+                config.setValue(argLineToSet + " " + current);
```

Comment:
Are you setting some bootclasspath in `argLineToSet`? If so, what happens if `current` already sets `some bootclasspath?